lib/TH/generic/THTensorRandom.c

```diff
@@ -70,6 +70,141 @@ void THTensor_(logNormal)(THTensor *self, THGenerator *_generator, double mean,
   TH_TENSOR_APPLY(real, self, *self_data = (real)THRandom_logNormal(_generator, mean, stdv););
 }
 
+
+void THTensor_(alias_multinomial_setup)(THTensor *probs, THLongTensor *J, THTensor *q)
```

Comment:
Can you make the name conforming to everything else in the library? Use e.g. `multinomialAliasSetup` and `multinomialAliasDraw`